dev/com.ibm.ws.jca_fat/fat/src/com/ibm/ws/jca/fat/app/JCATest.java

```diff
@@ -91,25 +86,6 @@ public static void setUp() throws Exception {
         ShrinkHelper.addDirectory(fvtapp_ear, "lib/LibertyFATTestFiles/fvtapp");
         ShrinkHelper.exportToServer(server, "apps", fvtapp_ear);
 
-        if (JakartaEE9Action.isActive()) {
-            /*
-             * Need to update the destination type of the topic to ensure it matches the Jakarta FQN.
-             */
-            ServerConfiguration clone = server.getServerConfiguration().clone();
```

Comment:
IMHO adding this kind of transformation to the repeat action implementation -- just after the features are swapped, before the updated configuration is written to file -- is a blazing fast, intuitive solution.  But solution could require specific information like config IDs in order to work correctly, so I didn't pursue it and don't have a strong opinion whether we should.